app/src/main/java/io/gnosis/safe/helpers/AddressHelper.kt

```diff
@@ -14,11 +14,12 @@ class AddressHelper
     suspend fun populateAddressInfo(
         addressView: TextView,
         imageView: BlockiesImageView?,
-        address: Solidity.Address
+        address: Solidity.Address,
+        chainPrefix: String?
```

Comment:
Would it make sense to have either `chainPrefix` be a part of `Solidity.Address` or have a `Safe.Address` type, that has both as fields? Or at least have it in `AddressInfo`?